### PR TITLE
eschema: Convert eschema grammar to SDL closer to DDL.

### PR DESCRIPTION
**Pull request by vpetrovykh**
The language used to defined eschema is currently very similar to DDL
and very different (case-sensitive, whitesapce-sensitive). Which means
that in principle it's a whole new language that needs to be documented
(by us) and learned (by the users). This is a non-trivial burden.
There's also some confusion as to which is primary source of truth - DDL
or eschema?

The new approach is to make the Schema Definition Language (SDL) to be
much more closely related to DDL. The rule of thumb is to make it so
that if you take a CREATE ... statement from DDL and strip out all
the CREATE and SET keywords you should end up with a valid SDL
declaration. The goal of DDL is to specify how to change the DB from
one state to the next, the goal of SDL is to describe the final state.
This is why SDL just mirrors CREATE and SET DDL since we're not
interested in the exact details of the steps.

The grammar has been stripped and moved to the EdgeQL grammar tree, but
the package is still called "eschema" and has a separate codegen and ast.

The command `edb gen-meta-grammars` now only has one valid argument
`edgeql`. However, we still keep two pygments grammars using it
(potentially useful if there's different quirks that need to be
accounted for).

Drop the parentheses from the `EXTENDING` list in both SDL and DDL.

Issue #498.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,5 +20,5 @@ postgres:
 pygments:
 	out=$$(edb gen-meta-grammars edgeql) && \
 		echo "$$out" > edb/edgeql/pygments/meta.py
-	out=$$(edb gen-meta-grammars edgeql eschema) && \
+	out=$$(edb gen-meta-grammars edgeql) && \
```
Comment (1st1) [inline Makefile]: We don't need eschema/pygments anymore

File: edb/eschema/codegen.py [hunk 2]
```diff
@@ -61,28 +63,29 @@ def _visit_specs(self, node):
...
             if getattr(node, 'properties', None):
-                self._visit_list(node.properties)
+                self.visit_list(node.properties, terminator=';')
             if getattr(node, 'attributes', None):
-                self._visit_list(node.attributes)
+                self.visit_list(node.attributes, terminator=';')
             if getattr(node, 'constraints', None):
-                self._visit_list(node.constraints)
+                self.visit_list(node.constraints, terminator=';')
             if getattr(node, 'policies', None):
```
Comment (elprans) [inline edb/eschema/codegen.py]: There are no 'policies' anymore.

File: edb/edgeql/parser/grammar/sdl.py [hunk 3]
```diff
@@ -0,0 +1,975 @@
...
+
+    def _block2(self, lbrace, sc1, cmdlist, sc2, rbrace):
+        self.val = cmdlist.val
+
+
+def commands_block(parent, *commands, opt=True):
+    if parent is None:
+        parent = ''
+
+    clsdict = collections.OrderedDict()
```
Comment (1st1) [inline edb/edgeql/parser/grammar/sdl.py]: No need to use OrderedDict since Python 3.6 (except for LRU caches)

File: edb/edgeql/pygments/meta.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # AUTOGENERATED BY EdgeDB WITH
 #     $ edb gen-pygments-grammars edgeql
 #
-# TIMESTAMP: 2019-03-12 19:46:27
+# TIMESTAMP: 2019-03-14 13:13:13
```
Comment (1st1) [inline edb/edgeql/pygments/meta.py]: No need for this change. In fact, I think this was a bad idea of mine to include the TIMESTAMP at all.  We can always use git to see when the file was last updated.  So please drop the TIMESTAMP comment generation from `edg gen-meta-grammars`

File: Makefile (hunk 1 above)
Comment (vpetrovykh) [reply Makefile]: Pygments dropped from code and make target.

File: edb/eschema/codegen.py (hunk 2 above)
Comment (vpetrovykh) [reply edb/eschema/codegen.py]: Dropped policies.

File: edb/edgeql/pygments/meta.py (hunk 4 above)
Comment (vpetrovykh) [reply edb/edgeql/pygments/meta.py]: Done

File: edb/edgeql/parser/grammar/sdl.py (hunk 3 above)
Comment (vpetrovykh) [reply edb/edgeql/parser/grammar/sdl.py]: Dropped.

**Pull request merged**